docs/examples/http_exceptions.py

```diff
@@ -0,0 +1,49 @@
+import json
+
+from nameko.web.handlers import HttpRequestHandler
+from werkzeug.wrappers import Response
+
+
+class ApplicationError(Exception):
+    pass
+
+
+class HttpError(Exception):
+
+    def __init__(self, payload, status_code=None):
```

Comment:
I simplified example of raising custom http exception. Adding status code in payload is a common practice (twilio, eventbrite, uber) but I removed it from the example to avoid any confusion.  
Please review.